spec/abstractops.html

```diff
@@ -1624,8 +1627,9 @@ <h1>
       <dd>It converts _argument_ to an integer representing its Number value (replacing *NaN* with 0), or throws a *RangeError* when that Number value is not a finite integral Number.</dd>
     </dl>
     <emu-alg>
+      1. If _argument_ is *undefined*, return 0.
```

Comment:
Same, we can remove this now, ToNumber(undefined) will return NaN which will be caught by IsIntegralNumber below.